ytools/y2tool/rubocop-0.71.0_yast_style.yml

```diff
@@ -0,0 +1,231 @@
+################################################################################
+#
+# This part contains the shared Rubocop configuration for SUSE projects. It is
+# maintained at https://github.com/SUSE/style-guides/blob/master/rubocop-suse.yml
+#
+# NOTE: some rules have been commented out, see the YaST specific changes
+#       at the end of the file!
+#
+################################################################################
+
+Layout/EndAlignment:
+  StyleGuide: https://github.com/SUSE/style-guides/blob/master/Ruby.md#lintendalignment
```

Comment:
The global SUSE style guide has been dropped, that page returns Error 404. We should remove all references from the config. (Even from the old one.)